runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -83,25 +83,25 @@ pub use proposal_types::{ProposalDetails, ProposalDetailsOf, ProposalEncoder};
 // 'Set working group mint capacity' proposal limit
 const WORKING_GROUP_MINT_CAPACITY_MAX_VALUE: u32 = 5_000_000;
 // Max allowed value for 'spending' proposal
-const MAX_SPENDING_PROPOSAL_VALUE: u32 = 5_000_000_u32;
+const MAX_SPENDING_PROPOSAL_VALUE: u32 = 50_000_000_u32;
 // Max validator count for the 'set validator count' proposal
 const MAX_VALIDATOR_COUNT: u32 = 300;
 // council_size min value for the 'set election parameters' proposal
-const ELECTION_PARAMETERS_COUNCIL_SIZE_MIN_VALUE: u32 = 4;
+const ELECTION_PARAMETERS_COUNCIL_SIZE_MIN_VALUE: u32 = 6;
 // council_size max value for the 'set election parameters' proposal
-const ELECTION_PARAMETERS_COUNCIL_SIZE_MAX_VALUE: u32 = 20;
+const ELECTION_PARAMETERS_COUNCIL_SIZE_MAX_VALUE: u32 = 40;
 // candidacy_limit min value for the 'set election parameters' proposal
 const ELECTION_PARAMETERS_CANDIDACY_LIMIT_MIN_VALUE: u32 = 50;
 // candidacy_limit max value for the 'set election parameters' proposal
-const ELECTION_PARAMETERS_CANDIDACY_LIMIT_MAX_VALUE: u32 = 100;
+const ELECTION_PARAMETERS_CANDIDACY_LIMIT_MAX_VALUE: u32 = 200;
 // min_voting_stake min value for the 'set election parameters' proposal
 const ELECTION_PARAMETERS_MIN_STAKE_MIN_VALUE: u32 = 1;
 // min_voting_stake max value for the 'set election parameters' proposal
 const ELECTION_PARAMETERS_MIN_STAKE_MAX_VALUE: u32 = 100_000_u32;
 // new_term_duration min value for the 'set election parameters' proposal
-const ELECTION_PARAMETERS_NEW_TERM_DURATION_MIN_VALUE: u32 = 14400;
+const ELECTION_PARAMETERS_NEW_TERM_DURATION_MIN_VALUE: u32 = 1;
```

Comment:
1 block ?

Reply:
Yeah, I considered doing this low value as the starting default.

Reply:
Makes sense now given the explanation here https://github.com/Joystream/joystream/pull/2307#discussion_r605058902